src/main/java/com/ashkan/moviesapi/controllers/ActorController.java

```diff
@@ -0,0 +1,35 @@
+package com.ashkan.moviesapi.controllers;
+
+import com.ashkan.moviesapi.entities.Actor;
+import com.ashkan.moviesapi.services.interfaces.ActorService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/actors")
+public class ActorController {
+    private ActorService actorService;
+
+    @Autowired
+    public ActorController(ActorService actorService) {
+        this.actorService = actorService;
+    }
+
+    @GetMapping
+    public List<Actor> findAll() {
+        return actorService.findAll();
+    }
+
+    @GetMapping("{Id}")
+    public Actor getActor(@PathVariable int Id) {
```

Comment:
Revisa por favor las buenas practicas para el nombrado de los paths de los endpoints, no es muy usual usar mayúsculas al iniciar el nombre de un sub-recurso, en tal caso el path de este endpoint tiene que ser: /{id}
Es posible me equivoque en ello, pero puedes ver que es lo más recomendable en este caso.

Reply:
Endpoints changed from {Id} to {id},meanwhile looking for rest best practices